src/TreeHelper/RobotOutOfFieldHelper.cpp

```diff
@@ -0,0 +1,35 @@
+//
+// Created by jessevw on 19.11.19.
+//
+#include <include/roboteam_ai/TreeHelper/RobotOutOfFieldHelper.h>
+#include <include/roboteam_ai/bt/composites/Sequence.hpp>
+#include <include/roboteam_ai/conditions/RobotOutside.h>
+#include <include/roboteam_ai/skills/gotopos/GTPSpecial.h>
+#include "bt/RoleDivider.h"
+#include "bt/Role.h"
+#include "skills/Attack.h"
+
+// Reference wrapper
```

Comment:
yeeaaah! std::reference_wrapper instead of pointers could go a long way :eyes: however, reference_wrapper is owning, so not sure.